cadvisor.go

```diff
@@ -34,8 +34,8 @@ import (
 
 	"crypto/tls"
 
-	"github.com/golang/glog"
-	"github.com/google/cadvisor/metrics"
+		"github.com/google/cadvisor/metrics"
```

Comment:
this looks strange.  I am surprised it passes go lint?  If you put a new line between these, does it go away?